tests/jeos/reset_locale.pm

```diff
@@ -0,0 +1,29 @@
+# SUSE's openQA tests
```

Comment:
This module cannot be de-coupled from jeos-firstboot. So it should run as post_run_hook in case of need